openwisp_radius/api/serializers.py

```diff
@@ -346,10 +346,23 @@ def validate_phone_number(self, phone_number):
             phone_number = None
         return phone_number
 
+    def validate_user_email(self, email):
+        email = email.lower()
```

Comment:
@sankha555 here email is the optional field. You should write something like:-
```suggestion
        if email:
            email = email.lower()
```